soroban-preflight/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "soroban-preflight"
+description = "Soroban transaction preflight computations."
+homepage = "https://github.com/stellar/rs-soroban-env"
+repository = "https://github.com/stellar/rs-soroban-env"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+version.workspace = true
+readme = "../README.md"
+edition = "2021"
+rust-version = "1.74"
```

Comment:
```suggestion
rust-version.workspace = true
```

Reply:
I cannot do this right now since there is no top-level version. I think that belongs to a separate PR

Reply:
There's a top level version in `main` now. Go for it 😄